2/README.md

```diff
@@ -0,0 +1,122 @@
+```
+shortname: 2/COSS
+name: Consensus-Oriented Specification System
+status: raw
+editor: Alberto Granzotto <alberto@bigchaindb.com>
+```
+
+This document describes a consensus-oriented specification system (COSS) for building interoperable technical specifications. COSS is based on a lightweight editorial process that seeks to engage the widest possible range of interested parties and move rapidly to consensus through working code.
+
+This specification is based on the original [Digistan 1/COSS](http://www.digistan.org/spec:1/COSS)
+
+
+## Change Process
+This document is governed by the [2/COSS](../2/README.md) (COSS).
+
+## Language
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119)
+
+## Goals
+The primary goal of COSS is to facilitate the process of writing, proving, and improving new technical specifications. A "technical specification" defines a protocol, a process, an API, a use of language, a methodology, or any other aspect of a technical environment that can usefully be documented for the purposes of technical or social interoperability.
+
+COSS is intended to above all be economical and rapid, so that it is useful to small teams with little time to spend on more formal processes.
+
+Principles:
+
+* We aim for rough consensus and running code.
+* Specifications are small pieces, made by small teams.
+* Specifications should have a clearly responsible editor.
+* The process should be visible, objective, and accessible to anyone.
+* The process should clearly separate experiments from solutions.
+* The process should allow deprecation of old specifications.
+
+Specifications should take minutes to explain, hours to design, days to write, weeks to prove, months to become mature, and years to replace.
+
+Specifications have no special status except that accorded by the community.
+
+## Architecture
+COSS is designed around fast, easy to use communications tools. Primarily, COSS uses a wiki model for editing and publishing specifications texts.
+
+* The *domain* is the conservancy for a set of specifications in a certain area.
+* Each domain is implemented as an Internet domain, hosting a wiki and optionally other communications tools.
+* Each specification is a set of wiki pages, together with comments, attached files, and other resources.
+* Important specifications may also exist as subdomains, i.e. child wikis.
+
+Individuals can become members of the domain by completing the necessary legal clearance. The copyright, patent, and trademark policies of the domain must be clarified in an Intellectual Property policy that applies to the domain.
```

Comment:
Is it related to the intellectual property of the proposal?
If so, [PEP says](https://www.python.org/dev/peps/pep-0001/#what-belongs-in-a-successful-pep):
> Copyright/public domain -- Each PEP must either be explicitly labeled as placed in the public domain (see this PEP as an example) or licensed under the Open Publication License [7].

[BIP says](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#BIP_licensing):
> New BIPs may be accepted with the following licenses. Each new BIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respective abbreviation given below.

What is fun:
- I don't find anything related to licensing for the Ethereum Improvement Proposal guidelines
- Python Enhancement Proposal guidelines suggest to use the Open Documentation License, while Bitcoin Improvement Proposal guidelines say that Open Documentation License is **not** an acceptable license.

Reply:
Can we do a bulk search for "domain" and replace all?

Reply:
EIPs must be public domain. It's the last, required section of each EIP: copyright assignment. A handy way to do that is to use a CC0 Public Domain Dedication - see https://creativecommons.org/publicdomain/zero/1.0/